configs/config.json.example

```diff
@@ -13,6 +13,9 @@
     "test": false,
     "location_cache": true,
     "distance_unit": "km",
+    "pokestop_cooldown": "300",
```

Comment:
@Viper896 you already have better parameters down here to do this: 

```
 "cooldown_min": 300,
 "cooldown_max": 900,
```

It should be clear in code that default cooldown is 5 minutes and we add a random delta because we want. 

I would even rename and simplify these keys to:

```
 "pokestop_cd_delta_min": 300,
 "pokestop_cd_delta_max": 600,
```

This would mean: for every pokestop spun sum randint(300, 600) to the server-given cooldown.